pennylane/tracker.py

```diff
@@ -72,7 +72,7 @@ def circuit(x):
     You can then access the tabulated information through ``totals``, ``history``, and ``latest``:
 
     >>> tracker.totals
-    {'batches': 2, 'simulations': 3, 'executions': 3, 'shots': 300}
+    {'batches': 2, 'simulations': 3, 'executions': 3, 'results': 1.18, 'shots': 300}
```

Comment:
Please check the rest of examples. It seems that:

1. There is an item missing in the docs when `tracker.history.keys()` is applied
2. Some of the printed numbers in `results` are different in both places that `results` is printed.